src/plugins/fixYoutubeEmbeds.desktop/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+
+export default definePlugin({
+    name: "FixYoutubeEmbeds",
+    description: "Fixes youtube embeds from being blocked",
```

Comment:
i think smth like that would be better
```suggestion
    description: "Bypasses youtube videos being blocked from display on Discord (for example by UMG)",
```